### PR TITLE
use master LB for cluster access

### PR DESCRIPTION
**Pull request by rimusz**
blocked by https://github.com/dcos/terraform-dcos/pull/29

### CODE REVIEW

**Comment by rimusz** [comment]
PR https://github.com/dcos/terraform-dcos/pull/29 got merged, can someone take a look to this PR, please?
cc @pires @spahl

File: Makefile [hunk 1]
```diff
@@ -83,6 +83,10 @@ kubectl-tunnel:
 		-N -L 9000:apiserver-insecure.kubernetes.l4lb.thisdcos.directory:9000 \
 		$(SSH_USER)@$(MASTER_IP)
 
+ui:
```
Comment (pires) [inline Makefile]: There are no docs referencing this new goal.

**Comment by rimusz** [comment]
The main purpose of this PR is to connect with `dcos` cli to master/s via masters LoadBalancer, instead using IP of one master.

**Comment by rimusz** [comment]
I usually leave commit squash for github :)

**Pull request merged**